tools/image_processing/imagej2/imagej2_analyze_particles_binary.xml

```diff
@@ -97,32 +88,55 @@ $circularity_max
             <option value="no" selected="True">No</option>
             <option value="yes">Yes</option>
         </param>
+        <param name="export_rois" type="boolean" truevalue="true" falsevalue="false" checked="false" label="Export ROIs coordinates" help="For example to upload to OMERO" />
     </inputs>
     <outputs>
         <data name="output" format_source="input">
             <filter>show != "Nothing"</filter>
        </data>
         <data format="tabular" name="results" label="${tool.name} (results) on ${on_string}">
-            <filter>show == "Nothing" or display_results_cond['display_results'] == "yes"</filter>
+            <filter>show == "Nothing" or display_results == "yes"</filter>
+        </data>
+        <data format="tabular" name="rois" label="${tool.name} (ROI coordinates) on ${on_string}">
```

Comment:
Same, maybe it is better to use "particles coordinates" instead of ROIs coordinates